docker/playground/compose.yml

```diff
@@ -78,3 +78,15 @@ services:
       - metastore
       - minio
       - zookeeper
+
+  kyuubi-client:
```

Comment:
it's unnecessary to create a new container as the client, we can reuse the server one.
```
docker exec -it kyuubi /opt/kyuubi/bin/beeline -u 'jdbc:hive2://0.0.0.0:10009/'
```